### PR TITLE
Feat: Already uploaded images(for edit purposes) 

### PR DESCRIPTION
**Pull request by sabrio**
Feature requested in #12 for edit purposes, also I added demo on the first page. 
Please review this PR and let me know what do you think. I/We need this feature immediately in. 
Thank you.

### CODE REVIEW

**Comment by UncleDave** [comment]
Looks good - thanks for this PR and the demo, could you please also update the readme accordingly?

**Comment by sabrio** [comment]
NP @UncleDave, yes ofc I can update it.

File: demo/src/app/demo/components/uploaded/uploaded.component.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+})
+export class UploadedExampleComponent {
+  myHeaders: { [name: string]: any } = {
+    'Authorization': 'MyToken',
+    'Another Header': 'AnotherValue'
+  };
+
+  images = [];
+
+  onRemoved(event){
```
Comment (aberezkin) [inline demo/src/app/demo/components/uploaded/uploaded.component.ts]: `){` => `) {` also :)

**Comment by aberezkin** [comment]
Also, I'm glad that you chose our lib for your first GitHub contribution :)

**Comment by UncleDave** [comment]
On the subject of code style, as raised by @aberezkin, there is an `.editorconfig` file in the repo - WebStorm supports this natively, and Visual Studio Code has a plugin for this although I'm not sure Visual Studio Code will match all of our styles correctly. If you're not using WebStorm I don't mind giving it a run through the autoformat after we merge this.

**Comment by sabrio** [comment]
@aberezkin thank you for your review&code style correction and I am also glad to help you guys improve this library because I found this library really useful and simple. This is what Autoformat did to the component 2a827436f1eaa508cf03b79cc3cdd0f3b2c29daa. I will revert it if needed. Let me know what do you think?

**Comment by UncleDave** [comment]
Fine by me.

**Comment by sabrio** [comment]
@UncleDave when this feature is going to be available for use in the package? Are you planning for pre-release so can use the development features and test them? smth like: 1.0.0-alpha.1

**Comment by UncleDave** [comment]
I have made that suggestion in #102, but I'm not the one who controls npm releases.

**Pull request merged**